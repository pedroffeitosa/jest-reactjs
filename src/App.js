import React from 'react';
import { Provider } from 'react-redux';

import store from './store';

import Todos from './Todos';


function App() {
  return (
    <Provider store={store}>
      <div className="App">
       <h1>Hello World!</h1>
       <Todos/>
      </div>
    </Provider>
  );
}

export default App;
