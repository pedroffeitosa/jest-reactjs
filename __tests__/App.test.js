import React from 'react';
import { shallowToJson } from 'enzyme-to-json';
import { shallow } from 'enzyme';
import App from '../src/App';

describe('Testing App Component',()=>{
  it('should render correctly', ()=> {
    const wrapper = shallow(<App/>);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  })
})