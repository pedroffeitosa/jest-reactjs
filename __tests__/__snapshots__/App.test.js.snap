// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing App Component should render correctly 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <div
    className="App"
  >
    <h1>
      Hello World!
    </h1>
    <Connect(Todos) />
  </div>
</Provider>
`;
